src/components/orders/DetailsTable/index.tsx

```diff
@@ -69,6 +79,7 @@ const tooltip = {
   from: 'The account address which signed the order.',
   to: 'The account address which will/did receive the bought amount.',
   hash: 'The onchain settlement transaction for this order. Can be viewed on Etherscan.',
+  appData: 'The app data document for this order',
```

Comment:
This tooltip is a bit short of info. 

@alfetopito does it make sense to include an external link/doc explaining more about this? do we have one?

Reply:
Well, there is this https://docs.cow.fi/front-end/creating-app-ids
Although it's more focused on creating the appIds.

Reply:
I think we could mention what this is intended to be, but too much detail will probably be detrimental.
Something along the lines of:

`The app data hash for this order. It can denote encoded metadata with info on the app, environment and more, although not all interfaces follow the same pattern. `Show more` will try to decode that information.`

Although, that might be too technical 😅